src/gravel/controllers/errors.py

```diff
@@ -0,0 +1,24 @@
+# project aquarium's backend
+# Copyright (C) 2021 SUSE, LLC.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+from typing import Optional
+
+
+class GravelError(Exception):
```

Comment:
(nit) if this is a "gravel" error, should it be in `src/gravel/errors.py`, rather than limiting it to controllers?